recipes/tbb/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# clang: error: invalid deployment target for -stdlib=libc++
+export MACOSX_DEPLOYMENT_TARGET=10.8
+
+make -j${CPU_COUNT}
```

Comment:
Please drop the `CPU_COUNT`. CIs are not always great about reporting this accurately.